nixos/tests/all-tests.nix

```
@@ -683,6 +683,7 @@ in {
   terminal-emulators = handleTest ./terminal-emulators.nix {};
   tiddlywiki = handleTest ./tiddlywiki.nix {};
   tigervnc = handleTest ./tigervnc.nix {};
+  timescaledb = handleTest ./timescaledb.nix {};
```

what do you think of renaming this to `timescaledb-toolkit`?
```suggestion
  timescaledb-toolkit = handleTest ./timescaledb-toolkit.nix {};
```

---

There was no test for timescaledb already, so this test is a very minimal test for both things, timescaledb and timescaledb-toolkit.
But I am not attached to the name. I will go ahead and change this.